examples/bell.py

```diff
@@ -10,13 +10,57 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+import cmath
+import math
+import time
+
+import numpy as np
 
 from braket.aws import AwsDevice
 from braket.circuits import Circuit
 
 device = AwsDevice("arn:aws:braket:::device/quantum-simulator/amazon/sv1")
 
 # https://wikipedia.org/wiki/Bell_state
-bell = Circuit().h(0).cnot(0, 1)
+
+bell = Circuit()
```

Comment:
This circuit, as well as the entire file, is now a misnomer

Reply:
Yep good catch